frontend/app/src/components/accounts/blockchain/Eth2Input.vue

```diff
@@ -102,6 +121,7 @@ watch(
         :disabled="disabled"
         :label="tc('eth2_input.public_key')"
         :error-messages="v$.publicKey.$errors.map(e => e.$message)"
+        @focus="removeError('publicKey')"
```

Comment:
@lukicenturi question can you remove this remove errors and the event handler for this,

Then go to the ValidatorAccountForm comment the code and change it so that it sets errorMessages?

```ts
set(errorMessages, { "ownershipPercentage": ["i have an error"] }
```

and play a bit in the form. Add any validator index press save get the error on ownershipPercentage, then go edit the ownership percentage and see if that clears the externalError and allows you to save again.

Reply:
I did a check and it seems that it still works.